src/components/Reports/GoalCalculator/CalculatorSettings/Steps/InformationStep/InformationStepForm/InformationStepPersonalForm.tsx

```diff
@@ -16,7 +16,13 @@ import { BenefitsPlan, Role } from './enums';
 import { locations } from './geographicAdjustments';
 import { ageOptions, familySizeOptions, tenureOptions } from './mockData';
 
-export const InformationStepPersonalForm: React.FC = () => {
+interface InformationStepPersonalFormProps {
+  isSpouse?: boolean;
+}
+
+export const InformationStepPersonalForm: React.FC<
+  InformationStepPersonalFormProps
+> = ({ isSpouse }) => {
```

Comment:
I think that last name is hidden for spouses. Also, should we make the label "Spouse First Name" for spouses?